db/seeds.rb

```diff
@@ -187,12 +190,14 @@
     main_color_hex: "343d46",
     overcast_url: "https://overcast.fm/itunes1006105326/developer-on-fire",
     android_url: "http://subscribeonandroid.com/developeronfire.com/rss.xml",
-    image: Rack::Test::UploadedFile.new(image_file, "image/jpeg")
+    image: Rack::Test::UploadedFile.new(image_file, "image/jpeg"),
+    published: true
   },
 ]
 
 podcast_objects.each do |attributes|
-  Podcast.create!(attributes)
+  podcast = Podcast.create!(attributes)
+  Podcasts::GetEpisodesWorker.perform_async(podcast_id: podcast.id)
```

Comment:
Would it be worth it to put a limit here as we do in the the `fetch.rake` file? otherwise it will fetch ALL episodes

```ruby
Podcasts::GetEpisodesWorker.perform_async(podcast_id: podcast.id, limit: 5)
```

Reply:
I assumed it would be useful to have a lot of episodes locally.

Reply:
Let's keep it like this, already approved after all ;)

Reply:
Just curious: what's the time impact on loading these?

Reply:
I've never tried to download them all before (keep in mind it's async so it's going to be done by sidekiq the first time is booted), but I just did for fun's sake:

```ruby
podcasts = Podcast.all
podcasts.each { |p| p.update(published: true) }

start_time = Time.current
podcasts.each { |p| Podcasts::GetEpisodesWorker.new.perform(podcast_id: p.id) }
end_time = Time.current - start_time
puts end_time
```

then

```shell
> time rails runner t.rb
17.968629
15.76s user 3.51s system 32.64s real 262996kB mem -- rails runner t.rb
```

Less than 20 seconds when done inline